DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/Index.cshtml

```diff
@@ -0,0 +1,28 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+@model CentreConfigurationViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/centreConfiguration.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Centre Configuration";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading" class="nhsuk-heading-xl">Centre Configuration</h1>
```

Comment:
Could we have h1 as "Centre configuration" to match capitalisation of other pages? 🙏